platform-configuration/src/main/java/org/codice/ddf/configuration/ConfigurationManager.java

```diff
@@ -151,6 +150,11 @@
     protected Map<String, String> configuration;
 
     /**
+     * The initial configuration values from blueprint.
+     */
+    protected Map<String, String> configurationProperties = new HashMap<>();
```

Comment:
It could, and probably should. We do have a semi-convention of using protected a lot to make future overriding easier, in order to avoid getting boxed in by API freezes. I don't think that applies here though.